setup.cfg

```diff
@@ -1,10 +1,66 @@
 [metadata]
-package_name = jwst
-description = JWST Pipeline
-author = STScI
-license = BSD
-edit_on_github = False
-github_project = spacetelescope/jwst
+name = jwst
+description = Python library for science observations from the James Webb Space Telescope
+long_description = file: README.md
+long_description_content_type = text/markdown
+author = JWST Pipeline developers
+license = BSD 3-Clause License
+url = https://github.com/spacetelescope/jwst
+classifiers =
+    Intended Audience :: Science/Research
+    Topic :: Scientific/Engineering :: Astronomy
+    License :: OSI Approved :: BSD License
+    Programming Language :: Python :: 3
+
+[options]
+zip_safe = False
+python_requires = >=3.6
+setup_requires =
+    setuptools_scm
+install_requires =
+    asdf>=2.5
+    astropy>=4.0
+    crds>=7.4.1.3
+    drizzle>=1.13
+    gwcs>=0.13.0
+    jsonschema>=3.0.1
+    numpy>=1.16
+    photutils>=0.7
+    pyparsing>=2.2
+    requests>=2.22
+    scipy>=1.1.0
```

Comment:
Current release is 1.5. Should we up this too, e.g. to 1.3? I haven't kept up with scipy dev recently.

Reply:
Same here.  It works, so while people should use the newer one, it is not required.